parts/k8s/addons/coredns.yaml

```diff
@@ -35,7 +39,7 @@ metadata:
     rbac.authorization.kubernetes.io/autoupdate: "true"
   labels:
     kubernetes.io/bootstrapping: rbac-defaults
-    addonmanager.kubernetes.io/mode: EnsureExists
+    addonmanager.kubernetes.io/mode: Reconcile
```

Comment:
I think we might want to keep this as EnsureExists, see https://github.com/Azure/aks-engine/pull/1401/files/37bb24f4666a6d4e4db36edb28dc29027dcade8d#r288814213